create.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Kdam+Thmor+Pro&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <main>
+    <body>
+      <h1>CREATE</h1>
+      <h2 class="create">ADD Question</h2>
+      <section>
+        <article>
+          This is wehre you can create queistions!
+          <form action="Create" method="get">
+            <label for="question">Question:</label>
+            <input type="text" id="Question" name="question" /> <br />
+            <br />
+            <label for="question">Question:</label>
+            <input type="text" id="Question" name="question" />
+          </form>
+        </article>
+      </section>
+      <br />
+      <br />
+      <section>
+        <article>
+          This is wehre you type in Answer!
+          <form action="Create" method="get">
+            <label for="Answer">Answer:</label>
```

Comment:
Don't use a capital letter for the values of your attributes like. For example: `for="answer"` and not `for="Answer"`. This is not common practice.